src/pages/CreatePost.js

```diff
@@ -1,8 +1,23 @@
+import { useNavigate } from "react-router-dom";
+
 import PostForm from "../components/PostForm";
 
 function CreatePost() {
+  const navigate = useNavigate();
+
   function newPostHandler(postData) {
-      
+    fetch(
+      "https://friendface-react-default-rtdb.europe-west1.firebasedatabase.app/posts.json",
+      {
+        method: "POST",
+        body: JSON.stringify(postData),
```

Comment:
You have content type json but doesn't stringify turn the object into a string?

Reply:
postData which is being parsed is a JS object, so I thought I needed to make it a string before making it a json - unless a simple JSON.parse() might suffice?

Reply:
just realised JSON.parse creates a json object, so that wouldn't work

Reply:
JSON stands for JavaScript Object Notation so I would have thought it would be interchangeable